modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
 [NOTE]
 Run performance and load testing on your Mule applications to determine the number of resources to allocate.
 
+== Internal Load Balancer Memory Allocation
```

Comment:
What does "Internal" apply to: "load balancer", "memory", "allocation", or "memory allocation"? I don't think what it applies to is clear.